Jenkinsfile

```diff
@@ -18,7 +18,7 @@ node('docker') {
         /* Checkout the latest LTS release so we don't need to build Jenkins to create the site */
         sh 'git clone https://github.com/jenkinsci/jenkins/ jenkins'
         dir ('jenkins') {
-            sh 'git checkout jenkins-2.46'
+            sh 'git checkout jenkins-2.78'
```

Comment:
I'm discovering this repo TBH. Care to fix/clarify the comment above this? 2.46 was an LTS, 2.78 is not, so merging as-is would make the comment go moot and very misleading to readers.

Reading the commit logs didn't help either. What is the intent here? How does it fail? I think we ought to store more things here about the failure for reference.